Week-2/Exercise 2.1/base.html

```diff
@@ -0,0 +1,165 @@
+            offered by Bikester, you’ll have a hassle-free experience. The spare
+            components used on your motorcycle are genuine and of high quality.
+          </p>
+        </section>
+      </article>
+
+      <article class="gir-one">
+        <div class="one-div kav">
+          <img
+            rel="icon"
+            type="image/png"
+            src="Image/temperature-check.png"
+            class="img-gir-one"
+          />
+          <h2>Staff Checking</h2>
+          <p>
+            All our staff health is monitored and body temperature is checked
+            prior to sending him on field
+          </p>
+        </div>
+
+        <div class="two-div kav">
+          <img
+            rel="icon"
+            type="image/png"
+            src="Image/mask.png"
+            class="img-gir-one"
+          />
+          <h2>Sanitization and Disinfecting</h2>
+          <p>
+            Bike surfaces and keys will be thoroughly sanitized when handing
+            over to you after servicing.
+          </p>
+        </div>
+
+        <div class="three-div kav">
+          <img
+            rel="icon"
+            type="image/png"
+            src="Image/doctor.png"
+            class="img-gir-one"
+          />
+          <h2>Contactless Working</h2>
+          <p>
+            Staff members will be in mask and gloves at all times while
+            interacting with you.
+          </p>
+        </div>
+      </article>
+
+      <div class="forms">
+        <h1 class="h-common">Enquire Form</h1>
+        <form action="#" method="get">
+          <input required type="text" placeholder="Name" class="fom" /><br />
+          <input
+            required
+            type="email"
+            placeholder="Email Id"
+            pattern="[a-zA-Z0-9.-_]{1,}@[a-zA-Z.-]{2,}[.]{1}[a-zA-Z]{2,}"
+            class="fom"
+          /><br />
+          <input
+            required
+            type="tel"
+            placeholder="Mobile No"
+            pattern="[6-9]{1}[0-9]{9}"
+            class="fom"
+          /><br />
+
+          <input  required type="text" placeholder="Address" class="fom" /><br />
+
+          <label>I Agree All Term and Condition </label
+          ><input  required type="checkbox" />
+          <br />
+          <!-- <input type="button" value="Submit" class="fom but" /> -->
+          <button class="fom but">Submit</button>
+        </form>
+      </div>
+
+      <footer class="foot">
+        <h2>All Right Reserved ©</h2>
+
+          <img
+            rel="icon"
+            type="image/png"
+            src="Image/facebook (1).png"
+            class="img-foot-one hov-one"
+          />
+          <img
+            rel="icon"
+            type="image/png"
+            src="Image/youtube.png"
+            class="img-foot-one hov-two"
+          />
+           <img
+            rel="icon"
+            type="image/png"
+            src="Image/twitter.png"
+            class="img-foot-one hov-three"
```

Comment:
does this code  has implementation of responsiveness and layout like horizontal or vertical

Reply:
No , The Footer Section Does Not have Resposiveness.